app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,64 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      erb :"/tweets/tweets"
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if !params["content"].empty?
+      @tweet = Tweet.create(content: params["content"], user_id: current_user.id)
+      redirect to '/tweets'
+    else
+      redirect to '/tweets/new'
+    end
+  end
+
+  delete '/tweets/:id' do
+    @tweet = Tweet.find_by(id: params[:id])
+    if current_user.id == @tweet.user_id
+      @tweet.delete
+    end
+    redirect to "/tweets"
+  end
+
+  patch '/tweets/:id' do
+    @tweet = Tweet.find_by(id: params[:id])
+    if !params["content"].empty?
```

Comment:
I would contend that you still want to double check the user's association to the tweet here. There are other ways to submit a PATCH/POST request than going through an HTML form...